jumpscale/servers/threebot/threebot.py

```diff
@@ -441,8 +441,13 @@ def delete(self, package_name):
                     self.threebot.gedis._system_actor.unregister_actor(actor)
 
             # unload chats
-            if package.chats_dir:
-                self.threebot.chatbot.unload(package.chats_dir)
+            try:
+                if package.chats_dir:
+                    self.threebot.chatbot.unload(package.chats_dir)
+            except:
+                j.core.logging.warning(
+                    f"Error during unloading the chats related to {package_name} package, maybe the package.toml file is not found."
```

Comment:
is that the cause? how the package got registered without validating the package first? also can we improve the phrasing a bit e.g "couldn't unload chats of {package_name} in path ...."

also I think we need to improve the handling of the unnnloadingn itself https://github.com/threefoldtech/js-sdk/blob/741ff0ba820bd72c7113f3f8aff7ffeb51d10f3e/jumpscale/packages/chatflows/actors/chatbot.py#L100, and in the loggin message at least we don't lose the information of the current exception / stacktrace, best we create an alert

Reply:
- I want to clarify something, I am handling an error occurred after adding the package successfully, like deletion of package.toml file then delete the package,
- This part raises an exception so early as it tries to get `chats_dir` so it tries to read the package.toml.
https://github.com/threefoldtech/js-sdk/blob/b00ce331f02068ee45a4010345956ca73dce0cba/jumpscale/servers/threebot/threebot.py#L445-L446

Reply:
Regarding the logging part. so you want me to show a warning and log the message with the actual exception, is it right?

Reply:
Exceptions shouldn't be hidden or masked, if we except any exception we catch it (so we have access to the raised exception and its data) then we decide how to handle it, in your case you decided logging is enough. then you can log with the original exception that you caught.

However, What I'm saying is the root cause appears to be in the `unload` method, and the fix should to be applied there

Reply:
![image](https://user-images.githubusercontent.com/46320202/91459837-6a4ece80-e887-11ea-9bae-1b097cc6fc23.png)

I have shown the exception to the user now 


regarding the `unload` method, it is actually not used anywhere in the code 
![image](https://user-images.githubusercontent.com/46320202/91459992-9a966d00-e887-11ea-95e9-36dffc0a0f1d.png)